templates/openj9/sso74-openj9-x509-https.json

```diff
@@ -369,6 +388,12 @@
                                 "secret": {
                                     "secretName": "sso-x509-jgroups-secret"
                                 }
+                            },
+                            {
+                                "name": "service-ca",
+                                "configMap": {
+                                    "configMapName": "${APPLICATION_NAME}-service-ca"
```

Comment:
@marun Replace `"configMapName"` with `"name"`, IOWs do:
```
[iankko@localhost tmp]$ diff -pruN sso74-x509-https.json.orig sso74-x509-https.json
--- sso74-x509-https.json.orig	2020-05-22 14:22:20.564266004 +0200
+++ sso74-x509-https.json	2020-05-22 14:25:35.693080510 +0200
@@ -392,7 +392,7 @@
                             {
                                 "name": "service-ca",
                                 "configMap": {
-                                    "configMapName": "${APPLICATION_NAME}-service-ca"
+                                    "name": "${APPLICATION_NAME}-service-ca"
                                 }
                             }
                         ]
```

In the current version the template, when calling `oc new-app --template=...` returns:
```
[iankko@localhost tmp]$ oc new-app --template=sso74-x509-https
--> Deploying template "iankko-inject-ca-bundle/sso74-x509-https" to project iankko-inject-ca-bundle

     Red Hat Single Sign-On 7.4 on OpenJDK (Ephemeral)
     ---------
     An example application based on RH-SSO 7.4 on OpenJDK image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso74-dev/docs.

     A new RH-SSO service has been created in your project. The admin username/password for accessing the master realm via the RH-SSO console is X86pvCyd/dMofOUHii7yIgsikM76ftX8ABG1HnUCf. The HTTPS keystore used for serving secure content, the JGroups keystore used for securing JGroups communications, and server truststore used for securing RH-SSO requests were automatically created via OpenShift's service serving x509 certificate secrets.

     * With parameters:
        * Application Name=sso
        * Custom RH-SSO Server Hostname=
        * JGroups Cluster Password=Gfire5WQssryMmI5LHG65VCpnswntTpy # generated
        * Datasource Minimum Pool Size=
        * Datasource Maximum Pool Size=
        * Datasource Transaction Isolation=
        * ImageStream Namespace=openshift
        * RH-SSO Administrator Username=X86pvCyd # generated
        * RH-SSO Administrator Password=dMofOUHii7yIgsikM76ftX8ABG1HnUCf # generated
        * RH-SSO Realm=
        * RH-SSO Service Username=
        * RH-SSO Service Password=
        * Container Memory Limit=1Gi

--> Creating resources ...
    configmap "sso-service-ca" created
    service "sso" created
    service "sso-ping" created
    route.route.openshift.io "sso" created
    error: DeploymentConfig.apps.openshift.io "sso" is invalid: [spec.template.spec.volumes[2].configMap.name: Required value, spec.template.spec.containers[0].volumeMounts[2].name: Not found: "service-ca"]
--> Failed
```

In the replaced form, it works correctly:
```
[iankko@localhost tmp]$ oc new-app --template=sso74-x509-https
--> Deploying template "iankko-inject-ca-bundle/sso74-x509-https" to project iankko-inject-ca-bundle

     Red Hat Single Sign-On 7.4 on OpenJDK (Ephemeral)
     ---------
     An example application based on RH-SSO 7.4 on OpenJDK image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso74-dev/docs.

     A new RH-SSO service has been created in your project. The admin username/password for accessing the master realm via the RH-SSO console is MGOBwlID/OkqS07eehu1cvdOmAPr3s6EJrqBhOMqq. The HTTPS keystore used for serving secure content, the JGroups keystore used for securing JGroups communications, and server truststore used for securing RH-SSO requests were automatically created via OpenShift's service serving x509 certificate secrets.

     * With parameters:
        * Application Name=sso
        * Custom RH-SSO Server Hostname=
        * JGroups Cluster Password=tpPjeJMQkWYTvtlmEo1A6mbAgR5oJRaw # generated
        * Datasource Minimum Pool Size=
        * Datasource Maximum Pool Size=
        * Datasource Transaction Isolation=
        * ImageStream Namespace=openshift
        * RH-SSO Administrator Username=MGOBwlID # generated
        * RH-SSO Administrator Password=OkqS07eehu1cvdOmAPr3s6EJrqBhOMqq # generated
        * RH-SSO Realm=
        * RH-SSO Service Username=
        * RH-SSO Service Password=
        * Container Memory Limit=1Gi

--> Creating resources ...
    configmap "sso-service-ca" created
    service "sso" created
    service "sso-ping" created
    route.route.openshift.io "sso" created
    deploymentconfig.apps.openshift.io "sso" created
--> Success
    Access your application via route 'sso-iankko-inject-ca-bundle.apps.rhsso43cluster.lab.rdu2.cee.redhat.com' 
    Run 'oc status' to view your app.
```

Reply:
Done.